### ISSUE TITLE
Add comma to numpy output

### ISSUE THREAD

**Issue by fanzfan** [haochengxia/vscode-pydata-viewer#28]
Adding commas as the separator between per elements makes it easy to copy from numpy outputs.  
I am glad to work on this.  
```python
# Before
Shape(10)
[1 2 3 4 5 6 7 8 9 10]
# If you copy the output, you cannot use it directly
arr = [1 2 3 4 5 6 7 8 9 10]    # illegal expression

# After
Shape(10)
[1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]    # legal, and easy to use
```

**Comment by haochengxia** [comment]
Sounds good! This feature would enhance the convenience of using this extension in a data science workflow.

**Comment by fanzfan** [comment]
Hi @haochengxia,
I found a compatibility problem, in "pydataPreview.js line 125", python does not support "&ensp;" space. 
We should use "&nbsp;", the example is as follows:
```python
# &ensp
arr = [1, 2, 3, 4, 5, 6, 7, 8]
# SyntaxError: invalid non-printable character U+2002

# &nbsp
arr = [1, 2, 3, 4, 5, 6, 7, 8]
# ok
```

**Comment by haochengxia** [comment]
Closed by Pull Request #29.
